tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -203,20 +273,47 @@ def clean_node(self, node):
 
     def current_assignment(self):
         with self.lock:
-            return self.assignment
+            return dict((handler.node, handler.current_assignment()) for handler in self.event_handlers.itervalues())
```

Comment:
Super minor, but you can also express this as
`return {handler.node: handler.current_assignment() for handler in self.event_handlers.values()}`